addons/website/static/src/js/editor/wysiwyg.js

```diff
@@ -205,6 +205,9 @@ Wysiwyg.include({
      * @returns {Promise}
      */
     _toggleMegaMenu: function (toggleEl) {
+        if (this.options.enableTranslation) {
+            return;
+        }
```

Comment:
@Goaman Seems ok but wanted to test: the editor panel does not load forever anymore indeed... but the mega menu is still not translatable :grimacing: (tested both on Firefox and Chrome, bugs in different ways)

Reply:
@qsm-odoo The PR that fix the megamenu is different: https://github.com/odoo/odoo/pull/86825

Reply:
@Goaman Ok, I tried regrouping both PR in local and test... still don't work: I can add text in terms to translate but not delete it. Tested both on Chrome and Firefox. Maybe it's worth regrouping all your work in a single PR to be sure that the combination works before merging? Not sure how I could r+ otherwise... :grimacing: Thanks :+1: